doc/distributions/nc_chi_squared.qbk

```diff
@@ -232,7 +232,7 @@ The PDF is computed directly using the relation:
 
 [equation nc_chi_squ_ref3]
 
-Where ['f(x; v)] is the PDF of the central __chi_squared_distrib and
+Where ['f(x;[nu])] is the PDF of the central __chi_squared_distrib and
 ['I[sub v](x)] is a modified Bessel function, see __cyl_bessel_i.
```

Comment:
@NAThompson I have doubt for the ['I[sub v](x)] it can be defined by other variables as well like ['I[sub (random variable)](x)] 
 I am not so sure please can you tell

Reply:
Yes, I see the issue. I'm going to merge your PR and I'll get this one.